README.md

```
@@ -93,4 +93,19 @@ kotlin-study/
 ## ⭐️ 참고사항
 - 모든 코드는 코틀린 스타일 가이드를 따라 작성할 것
 - 궁금한 점이 있다면 PR에 코멘트로 남길 것
-- 매일 학습 내용을 TIL로 정리할 것
\ No newline at end of file
+- 매일 학습 내용을 TIL로 정리할 것
+
+# 학습 기록 
+## [2024-12-16]
+- git의 구조와 git의 사용법
+- 삽질한 내용은 topics/git.md에 작성.
+#### 차후 찾아볼 내용 
```

Markdownlint (0.37.0)

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)